tck/old-tck-selenium/commandLink/src/main/java/com/sun/ts/tests/jsf/spec/render/commandlink/SimpleActionListener.java

```diff
@@ -31,11 +31,15 @@
  import jakarta.faces.event.ActionListener;
  import jakarta.servlet.http.HttpServletResponse;
  
- @jakarta.inject.Named("ActionListener") @jakarta.enterprise.context.SessionScoped
+ @jakarta.inject.Named("ActionListener") @jakarta.enterprise.context.RequestScoped
```

Comment:
When I tested this app manually, the result was already set in the later test attempts. So, I changed to RequestScoped. I can change it back, but it shouldn't affect the test.